### PR TITLE
Add LightGBM Slack team to README.md

### PR DESCRIPTION
**Pull request by henry0312**
- Add public Slack team
- Use Heroku app to send invitation (like https://github.com/fchollet/keras)
  - https://github.com/rauchg/slackin

I don't know why https://github.com/Microsoft/LightGBM/commit/a9258fe2c96994516df92488f5824943b373b386 removed Slack team information, but many open source comunity use Slack for user community to ask questions and join the development discussion, and there is no reason for us not use it.

See also https://github.com/Microsoft/LightGBM/issues/595

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ LightGBM, Light Gradient Boosting Machine
 [![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/Microsoft/LightGBM/blob/master/LICENSE)
 [![Python Versions](https://img.shields.io/pypi/pyversions/lightgbm.svg)](https://pypi.python.org/pypi/lightgbm)
 [![PyPI Version](https://badge.fury.io/py/lightgbm.svg)](https://badge.fury.io/py/lightgbm)
+[![Slack](https://lightgbm-slack-autojoin.herokuapp.com/badge.svg)](https://lightgbm-slack-autojoin.herokuapp.com/)
```
Comment (StrikerRUS) [inline README.md]: Could you please move Slack badge to gitter one (or gitter to Slack)?

Comment (henry0312) [reply README.md]: Did you mean,

```diff
diff --git a/README.md b/README.md
index f107148..23bcfd9 100644
--- a/README.md
+++ b/README.md
@@ -1,7 +1,6 @@
 LightGBM, Light Gradient Boosting Machine
 =========================================

-[![Join the chat at https://gitter.im/Microsoft/LightGBM](https://badges.gitter.im/Microsoft/LightGBM.svg)](https://gitter.im/Microsoft/LightGBM?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Build Status](https://travis-ci.org/Microsoft/LightGBM.svg?branch=master)](https://travis-ci.org/Microsoft/LightGBM)
 [![Windows Build Status](https://ci.appveyor.com/api/projects/status/1ys5ot401m0fep6l/branch/master?svg=true)](https://ci.appveyor.com/project/guolinke/lightgbm/branch/master)
 [![Documentation Status](https://readthedocs.org/projects/lightgbm/badge/?version=latest)](https://lightgbm.readthedocs.io/)
@@ -9,6 +8,7 @@ LightGBM, Light Gradient Boosting Machine
 [![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/Microsoft/LightGBM/blob/master/LICENSE)
 [![Python Versions](https://img.shields.io/pypi/pyversions/lightgbm.svg)](https://pypi.python.org/pypi/lightgbm)
 [![PyPI Version](https://badge.fury.io/py/lightgbm.svg)](https://badge.fury.io/py/lightgbm)
+[![Join the chat at https://gitter.im/Microsoft/LightGBM](https://badges.gitter.im/Microsoft/LightGBM.svg)](https://gitter.im/Microsoft/LightGBM?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Slack](https://lightgbm-slack-autojoin.herokuapp.com/badge.svg)](https://lightgbm-slack-autojoin.herokuapp.com/)
```

right?

Comment (StrikerRUS) [reply README.md]: Yeah!

Comment (henry0312) [reply README.md]: updated

**Pull request merged**